tqdm/_tqdm.py

```diff
@@ -190,7 +190,7 @@ def status_printer(file):
 
         def fp_write(s):
             fp.write(_unicode(s))
-            fp_flush()
+            fp.flush()
```

Comment:
didn't find def fp_flush() and I worked before with this kind of code so I thought that it should be fp.flush()